JSONPlaceholderAPIClient/APIClient.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Net;
+
+namespace JSONPlaceholderAPIClient
+{
+    public class ApiClient
+    {
+        public static HttpClient Client = new HttpClient();
+
+        public static void Put()
```

Comment:
As far as you want ApiClient class to have only static method you can put initialization into static constructor